pkg/controller/unifiedpushserver/unifiedpushserver_controller.go

```diff
@@ -265,7 +329,7 @@ func (r *ReconcileUnifiedPushServer) Reconcile(request reconcile.Request) (recon
 	}
 
 	//#region AMQ resource reconcile
-	if instance.Spec.UseMessageBroker {
+	if instance.Spec.UseMessageBroker && canWatchEnmasse(r.config) {
```

Comment:
Is it right to just ignore the spec when we can't create a watch? We can still create and use the resources, even if we're not going to watch them, right? I haven't fully thought this through, so I might well be wrong :)

Reply:
Well creating the child address causes the operator to start watching those resources.  I'm not really sure where, but the errors star appearing and the operator goes into an immediate infinite reconcile loop.  Since that behavior will bog down a enmasse server pre 0.31 I thought it was best to just disable it.